CHANGES.md

```diff
@@ -28,6 +28,11 @@ development source code and as such may not be routinely kept up to date.
   Batch job.
   ([#460](https://github.com/nextstrain/cli/pull/460))
 
+* When Nextstrain CLI detects that it needs to be updated, it will now
+  output the instructions on how to do any update after the information
+  for which packages were updated.
```

Comment:
Suggestion to clarify since "update" is used to refer to two different things, and "packages" doesn't apply to runtimes other than Conda or commands like `nextstrain check-setup`.

```suggestion
* When Nextstrain CLI detects that a new version of itself is available, it
  will now output the instructions on how to upgrade to that new version after
  (instead of before) the normal output from the running command.  This makes
  it easier to find and follow the instructions when output is long.
```